common/views/components/Buttons/ButtonButton/README.md

```diff
@@ -0,0 +1,10 @@
+### Primary button
+Main call to action on the page, e.g. 'Booking' or 'Sign up' - may include an icon.
+
+### Secondary button
+Used alongside the primary button as a supporting action, e.g. 'Cancel'.
```

Comment:
Do we have a rule here of that we never use this without a primary button?
If so, we can codify and demonstrate this in cardigan.

ping @Heesoomoon